### PR TITLE
Add skeleton for HHS facilities indicator

### LINKED ISSUE #627: Add data source for HHS hospitalizations using per-facility Epidata dataset
We are expediting creation of a new COVIDcast source+signals based on the following dataset:
https://healthdata.gov/dataset/covid-19-reported-patient-impact-and-hospital-capacity-facility

The raw data is expected to be served from its own Epidata endpoint shortly.

**Data details**
 
This data encodes hospital admissions on a per-facility basis, making it straightforward to aggregate to all COVIDcast geo levels.

Data is aggregated on a weekly basis, **Friday-Thursday** (note this is different from the standard epiweek definition, Sunday-Saturday)

The signals we want to compute for COVIDcast are:
- [ ] "ground truth" hospital admissions (`previous_day_admission_adult_covid_confirmed_7_day_sum` + `previous_day_admission_pediatric_covid_confirmed_7_day_sum`)
- [ ] ??? TBD

### PR DESCRIPTION
**Pull request by chinandrew**
### Description
Add the template and some placeholder information for the new HHS indicator. No code runs yet.

### Changelog
Itemize code/test/documentation changes and files added/removed.
- Add placeholder test and constants (exact signals are still TBD)
- Add some commented pseudocode for what will run
- Comment out some relevant imports and add to setup.py, also mainly as a placeholder
- Enable CI on this folder

### Fixes 
- Fixes #627

### CODE REVIEW

File: hhs_facilities/delphi_hhs_facilities/__init__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+"""Module to pull and clean indicators from the XXXXX source.
```
Comment (krivard) [inline hhs_facilities/delphi_hhs_facilities/__init__.py:2]: Fill in placeholders

File: hhs_facilities/delphi_hhs_facilities/__main__.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+"""Call the function run_module when executed.
+
+This file indicates that calling the module (`python -m MODULE_NAME`) will
```
Comment (krivard) [inline hhs_facilities/delphi_hhs_facilities/__main__.py:4]: Fill in placeholders

File: hhs_facilities/delphi_hhs_facilities/run.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""Functions to call when running the function.
+
+This module should contain a function called `run_module`, that is executed
+when the module is run with `python -m MODULE_NAME`.
```
Comment (krivard) [inline hhs_facilities/delphi_hhs_facilities/run.py:5]: Fill in placeholders

File: hhs_facilities/setup.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    version="0.1.0",
+    description="HHS hospital level indicators",
+    author="",
+    author_email="",
+    url="https://github.com/cmu-delphi/covidcast-indicators",
+    install_requires=required,
+    classifiers=[
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Developers",
+        "Programming Language :: Python :: 3.7",
```
Comment (krivard) [inline hhs_facilities/setup.py:27]: This should by 3.8 (gotta fix that template someday)

File: hhs_facilities/README.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+This command will install the package in editable mode, so you can make changes that
+will automatically propagate to the installed package. 
+
+All of the user-changable parameters are stored in `params.json`. To execute
+the module and produce the output datasets (by default, in `receiving`), run
+the following:
+
+```
+env/bin/python -m delphi_NAME
```
Comment (krivard) [inline hhs_facilities/README.md:25]: Fill in placeholders

File: hhs_facilities/README.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+command from this directory:
+
+```
+make test
+```
+
+To run individual tests, run the following:
+
+```
+(cd tests && ../env/bin/pytest <your_test>.py --cov=delphi_NAME --cov-report=term-missing)
```
Comment (krivard) [inline hhs_facilities/README.md:56]: Fill in placeholders

**Review by krivard — CHANGES_REQUESTED** [review]
one of these days we'll write a script to fill in placeholders

**Comment by chinandrew** [comment]
Thanks for pointing all these out, I've updated them now

**Review by krivard — APPROVED** [review]
🎉

**Pull request merged**